teams/nomis/pipelines/jumpserver/components/jumpserver/jumpserver.yml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
it's a long path to get to the jumpserver component, I am thinking, can we keep it centrally and list it to the trigger path in the GH workflow, separately to the main jumpserver directory

Reply:
I thought this would be the way we'd go, all components kept centrally but it would require some redoing of the workflow file which I think we'll be looking at more systematically next sprint.

Reply:
The module as written also expects them to be in subdirectories at the moment so that will need a bit of rework as well.